native/cocos/2d/renderer/RenderDrawInfo.h

```diff
@@ -289,8 +287,8 @@ class RenderDrawInfo final {
         scene::Model* _model;
         uint8_t* _sharedBuffer;
     };
-    gfx::InputAssemblerInfo* _iaInfo{nullptr};
-    ccstd::vector<gfx::InputAssembler*>* _iaPool{nullptr};
+    gfx::InputAssemblerInfo _iaInfo;
```

Comment:
No need to cache iaInfo in RenderDrawInfo，just create it in `initIAInfo`

Reply:
I tried, but I need a container to hold the pointer of vertexbuffers and indexbuffer.

Reply:
Oh, use smart_ptr instead of RenderDrawInfo may be more appropriate.